### PR TITLE
feat: add figma typographies variants

### PR DESCRIPTION
**Pull request by Reynaldoqsbol**
image

### CODE REVIEW

File: src/components/atoms/typography/ms-typography.styles.ts [hunk 1]
```diff
@@ -1,43 +1,31 @@
-import { css } from 'lit'
-
-import { setFont } from '~/utils'
-
-//@TODO: modify styles with var on style-dictionary & theme
```
Comment (cmermoz) [inline src/components/atoms/typography/ms-typography.styles.ts:5]: I think this comment can be erased now

File: src/components/atoms/typography/ms-typography.styles.ts [hunk 2]
```diff
@@ -1,43 +1,31 @@
...
-  p {
-    ${setFont(17, 24)}
-    font-weight: inherit;
+  :host > *:first-child {
     margin: 0;
+    box-sizing: border-box;
+    color: ${getThemeColor('gray-900')};
   }
+  ${unsafeCSS(typographyStylesGenerator())}
 `
```
Comment (cmermoz) [inline src/components/atoms/typography/ms-typography.styles.ts:31]: Just to understand, now we'll call this method every time we need to use the typographies' styles?

File: src/components/atoms/typography/ms-typography.styles.ts (hunk 1 above)
Comment (Reynaldoqsbol) [reply src/components/atoms/typography/ms-typography.styles.ts:5]: It's already removed

File: src/components/atoms/typography/ms-typography.styles.ts (hunk 2 above)
Comment (Reynaldoqsbol) [reply src/components/atoms/typography/ms-typography.styles.ts:31]: Nop, if you want to use typographies, u should use typography component `<ms-typography>`
eg:
```
<button><ms-typography variant="heading-2">Boton</ms-typography></button>
```
and also if you want to use the values of css variables, there are css variables to get those values => `--heading-medium-h1-font-size`

**Pull request merged**